From 3db7c78aee06cba3f7e03d11d2b467860050d533 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:08:24 -0400
Subject: [powerpc] numa: Fix decimal permissions

Message-id: <1410545655-205645-276-git-send-email-dzickus@redhat.com>
Patchwork-id: 94545
O-Subject: [RHEL7 PATCH 275/626] powerpc/numa: Fix decimal permissions
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 316d718827a9b546a95c50af5a4e3c2c8b953eee
Author: Joe Perches <joe@perches.com>
Date:   Tue Jan 28 10:22:22 2014 -0800

    powerpc/numa: Fix decimal permissions

    This should have been octal.

    Signed-off-by: Joe Perches <joe@perches.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/mm/numa.c b/arch/powerpc/mm/numa.c
index 8e01af2..3771c2a 100644
--- a/arch/powerpc/mm/numa.c
+++ b/arch/powerpc/mm/numa.c
@@ -1800,7 +1800,7 @@ static const struct file_operations topology_ops = {
 static int topology_update_init(void)
 {
  start_topology_update();
- proc_create("powerpc/topology_updates", 644, NULL, &topology_ops);
+ proc_create("powerpc/topology_updates", 0644, NULL, &topology_ops);
 
  return 0;
 }
-- 
1.7.1